data.sql

```diff
@@ -46,4 +46,42 @@ SET owner_id =
     WHEN name IN ('Devimon', 'Plantmon') THEN (SELECT id FROM owners WHERE full_name = 'Bob')
     WHEN name IN ('Charmander', 'Squirtle', 'Blossom') THEN (SELECT id FROM owners WHERE full_name = 'Melody Pond')
     WHEN name IN ('Angemon', 'Boarmon') THEN (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
-  END;
\ No newline at end of file
+  END;
+
+-- Inserting data in the vets table
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES
+  ('William Tatcher', 45, '2000-04-23'),
+  ('Maisy Smith', 26, '2019-01-17'),
+  ('Stephanie Mendez', 64, '1981-05-04'),
+  ('Jack Harkness', 38, '2008-06-08');
+
+-- Inserting data in the specializations table
+INSERT INTO specializations (vet_id, species_id)
+VALUES
+  ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon')),
+  ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon')),
+  ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon')),
+  ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));
+  
+-- Inserting data in the visits table
+INSERT INTO visits (animal_id, vet_id, visit_date) VALUES
+(1, 1, '2020-05-24'),
+(1, 3, '2020-07-22'),
+(2, 4, '2021-02-02'),
+(7, 2, '2020-01-05'),
+(7, 2, '2020-03-08'),
+(7, 2, '2020-05-14'),
+(3, 3, '2021-05-04'),
+(8, 4, '2021-02-24'),
+(4, 1, '2020-08-10'),
+(4, 2, '2021-04-07'),
```

Comment:
- [ ] The animal visited here is `Plantmon` with animal_id `6`. The animal with id 4 is Devimon. Kindly fix this, please.
- [ ] ALso, you forgot to add one more visit to plantmon by Maisy Williams on `2019-12-21`.
![image](https://github.com/ernestkorir/vet_clinic/assets/94551484/81f408e2-3bed-40bd-98a3-e369959d19d3)